README.md

```diff
@@ -1,4 +1,375 @@
-# GrowthBook SDK Flutter.
+![](https://docs.growthbook.io/images/hero.png)
 
-GrowthBook SDK for flutter.
+# Growth Book - SDK
```

Comment:
1. GrowthBook has no space. (this is through out this document)
2. Removing the flutter from the name - is that on purpose? Just wondering if people will know what it's for or how it is different than the other SDKs.